pkg/stanza/operator/input/udp/udp.go

```diff
@@ -113,12 +108,16 @@ func (c Config) Build(logger *zap.SugaredLogger) (operator.Operator, error) {
 		resolver = helper.NewIPResolver()
 	}
 
-	if c.AsyncConfig == nil {
-		c.AsyncConfig = NewAsyncConfig()
-	}
-
-	if c.AsyncConfig.Readers <= 0 {
-		return nil, fmt.Errorf("async readers must be greater than 0")
+	if c.AsyncConfig != nil {
+		if c.AsyncConfig.Readers <= 0 {
+			c.AsyncConfig.Readers = 1
```

Comment:
Can we move these default values into constants? They don't need to be exported, but it would add clarity.